src/status_im/contexts/keycard/migrate/re_encrypting/events.cljs

```diff
@@ -27,5 +27,8 @@
              :callback
              (fn [{:keys [error]}]
                (if (string/blank? error)
-                 (rf/dispatch [:navigate-to :screen/keycard.migrate.success])
-                 (rf/dispatch [:navigate-to :screen/keycard.migrate.fail])))}]]})))
+                 (do (rf/dispatch [:navigate-to :screen/keycard.migrate.success])
+                     (rf/dispatch [:centralized-metrics/track :metric/keycard-migration-succeeded]))
```

Comment:
I did something similar recently, but opted to track the aliased navigation event in `tracking/track-view-id-event`. No right or wrong.

https://github.com/status-im/status-mobile/blob/949e78a17e7a7a9367ab264b09b34c0b88d51df9/src/status_im/contexts/centralized_metrics/tracking.cljs#L55-L56